external/model-preparation-algorithm/configs/detection/mobilenetv2_atss_cls_incr/tile_pipeline.py

```diff
@@ -0,0 +1,89 @@
+dataset_type = "CocoDataset"
+
+img_size = (992, 736)
+
+tile_cfg = dict(
+    tile_size=400, min_area_ratio=0.9, overlap_ratio=0.2, iou_threshold=0.45, max_per_img=1500, filter_empty_gt=True
+)
+
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255, 255, 255], to_rgb=True)
```

Comment:
Why image normalize config is different with yolox? 
`img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)`

Reply:
This is the default setting for ATSS. But I'm guessing these were the normalise values that were used in training the pre-trained COCO model.